cryodrgn/dataset.py

```diff
@@ -1,9 +1,15 @@
 import numpy as np
+try:
+    import cupy as cp
+except:
+    cp = None
+
 import torch
 from torch.utils import data
 import os
 import multiprocessing as mp
 from multiprocessing import Pool
+from tqdm.auto import tqdm
```

Comment:
I would say let's keep `tqdm` out of this PR, since so far `cryoDRGN` doesn't need it directly (though some of its dependencies might). In general I like `tqdm` and there's quite a few places where we can use it, but we'll tackle it as a separate PR in the future.